lib/screens/scholarship_screen/scholarship_mobile.dart

```diff
@@ -53,123 +44,153 @@ class _ScholarshipMobileState extends State<ScholarshipMobile> {
+          style: TextStyle(
             color: Colors.white,
           ),
-          onPressed: (){
-            try{
-              openUrl(ScholarshipService.url);
-            }catch(e){
-            }
-          },
         ),
-      ],
-    );
+        actions: <Widget>[
+          IconButton(
+            padding: const EdgeInsets.all(5.0),
+            icon: Icon(Icons.search),
+            color: Colors.white,
+            onPressed: () {
+              setState(() {
+                isSearching = true;
+              });
+            },
+          ),
+          IconButton(
+            icon: Icon(
+              Icons.public,
+              color: Colors.white,
+            ),
+            onPressed: () {
+              try {
+                openUrl(ScholarshipService.url);
+              } catch (e) {}
+            },
+          ),
+        ],
+      );
+    }
+
+    return isSearching ? buildSearchBar() : buildRegularAppBar();
   }
 
-  Widget _buildTextField(){
+  Widget _buildTextField() {
     return TextField(
+      autofocus: true,
       focusNode: textfocus,
       controller: _controller,
       style: TextStyle(
-        color: Colors.grey.shade600,
+        color: Colors.white70,
       ),
       decoration: InputDecoration(
         border: OutlineInputBorder(
           borderSide: const BorderSide(
             width: 0,
             style: BorderStyle.none,
           ),
-          borderRadius: BorderRadius.all(const Radius.circular(70)),
-        ),
-        contentPadding: EdgeInsets.symmetric(
-          vertical: 2,
-          horizontal: 15,
         ),
         hintText: "Search",
-        hintStyle: TextStyle(color: Colors.grey.shade600,),
-        suffixIcon: Icon(
+        hintStyle: TextStyle(
+          color: Colors.white24,
+        ),
+        prefixIcon: Icon(
           Icons.search,
-          color: Colors.grey.shade600,
         ),
-        fillColor: Colors.grey.shade200,
-        filled: true,
       ),
-      onChanged: (query){
-        _debouncer.run((){
+      onChanged: (query) {
+        _debouncer.run(() {
           currentText = query;
           symbol.search(query);
         });
       },
-
-      onSubmitted: (query){
-        _debouncer.run((){
+      onSubmitted: (query) {
+        _debouncer.run(() {
           currentText = query;
           symbol.search(query);
         });
       },
     );
   }
 
-  Widget _buildTempList(List<Scholarship> lst){
-    
-    Widget _buildCard(int index){
+  Widget _buildTempList(List<Scholarship> lst) {
+    Widget _buildCard(int index) {
       return ScholarCard(scholarship: lst[index]);
     }
 
-    Widget _buildNoResults(){
+    Widget _buildNoResults() {
       return Center(
         child: Text(
-          "No Search Results", 
+          "No Search Results",
           style: TextStyle(
-            color: Colors.grey.shade400, 
+            color: Colors.grey.shade400,
             fontSize: 18,
             fontFamily: "Monsterrat",
```

Comment:
I think you forgot to remove this fontfamily declaration

Reply:
Changed as well